src/validators/index.ts

```diff
@@ -42,6 +42,16 @@ export const ArticleValidator = new Validator(
   },
 );
 
+export const CommentValidator = new Validator({
+  name: 'content',
+  location: 'body',
+  type: 'string',
+  require: true,
+  validator: (v: string) => v.length < bodyLength,
+  postProcess: (v: string) => filterXSS(v),
```

Comment:
We will remove `postProcess: (v: string) => filterXSS(v)` data should be store just like it is.

Reply:
Got it!